lib/web/apiserver.go

```diff
@@ -740,7 +740,7 @@ func defaultAuthenticationSettings(ctx context.Context, authClient auth.ClientI)
 
 func (h *Handler) ping(w http.ResponseWriter, r *http.Request, p httprouter.Params) (interface{}, error) {
 	var err error
-
+	fmt.Println("HONK HONK HONK", r.Header)
```

Comment:
if we're going to honk we should at least do it through a logger :slightly_smiling_face:

Reply:
ahah, whoops